<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="BDC4-D-1SF6N62" name="seleniumadvanced.demos.seleniumday1.NewTest" tests="7" failures="0" timestamp="08 Feb 2019 09:52:17 GMT" time="25.028" errors="1">
  <testcase name="f" time="0.000" classname="seleniumadvanced.demos.seleniumday1.NewTest">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="7.870" classname="seleniumadvanced.demos.seleniumday1.NewTest"/>
  <testcase name="f" time="9.187" classname="seleniumadvanced.demos.seleniumday1.NewTest"/>
  <testcase name="beforetest" time="0.019" classname="seleniumadvanced.demos.seleniumday1.NewTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at seleniumadvanced.demos.seleniumday1.NewTest.beforetest(NewTest.java:53)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- beforetest -->
  <testcase name="f" time="0.000" classname="seleniumadvanced.demos.seleniumday1.NewTest">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.000" classname="seleniumadvanced.demos.seleniumday1.NewTest">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="7.952" classname="seleniumadvanced.demos.seleniumday1.NewTest"/>
</testsuite> <!-- seleniumadvanced.demos.seleniumday1.NewTest -->
